MANIFEST.in

```diff
@@ -0,0 +1,6 @@
+
+# These get included because they are tracked by git, so explicitly exclude
```

Comment:
Can you add comments, what this file is for?

Reply:
Added